lib/admin-page.php

```diff
@@ -607,8 +607,10 @@ function classicpress_check_can_migrate() {
 	}
 
 	// Check: Supported PHP version
-	$php_version_min = '5.6';
-	if ( version_compare( PHP_VERSION, $php_version_min, 'lt' ) ) {
+	if ( isset( $parameters['php'] ) &&
+		version_compare( PHP_VERSION, $parameters['php']['min'], 'lt' ) ||
+		version_compare( PHP_VERSION, $parameters['php']['max'], 'gt' )
+	) {
```

Comment:
Indentation change and adding parentheses to make the logical order clearer (not strictly necessary, but easier to read and less likely to be accidentally broken later on):

```suggestion
	if (
		isset( $parameters['php'] ) && (
			version_compare( PHP_VERSION, $parameters['php']['min'], 'lt' ) ||
			version_compare( PHP_VERSION, $parameters['php']['max'], 'gt' )
		)
	) {
```

Reply:
Although, what should really happen if `$parameters['php']` is not set? As written, the PHP version check would "pass", which doesn't seem right.

`! isset( $parameters['php'] )` would be possible if we released the new version of the migration plugin _before_ deploying the API endpoint, which we shouldn't do. Is there another reason to leave this check in place?